RELEASE_NOTES.md

```diff
@@ -6,9 +6,10 @@ If your chain's state has coin metadata, an issue has been discovered in the ban
 
 * If you are planning to migrate to v0.46, please use `v0.46.5`. All releases prior to `v0.46.5`, **must NOT be used**. All previous version of `v0.46` are retracted.
```

Comment:
ah I just left a comment in your other pr around expanding on what retract means. could we add it here.

Reply:
Fixed!